src/Auth.php

```diff
@@ -40,7 +40,7 @@ public static function get_secret_key() {
 	 * @throws Exception
 	 * @since 0.0.1
 	 */
-	public static function login_and_get_token( $username, $password ) {
+	public static function login_and_get_token( $username, $password, $gRecaptchaResponse ) {
```

Comment:
`$gRecaptchaResponse` needs a default value otherwise it will throw an exception everytime it's value is `null`.
An `@param` entry in the docblock upon would be nice too. 👌🏿